### ISSUE TITLE
Skin lesion regresses spontaneously

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9998]
**Preferred term label:**
Skin lesion regresses spontaneously

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Keratoacanthoma

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
It seems it might be better to describe the resolution of lesions separately, using Phenopacket schema for instance, since for most skin conditions, it may or may not regress but it is hard to observe this directly...

**Comment by MickeySegal** [comment]
We also use this for:

- Yaws: https://www.ncbi.nlm.nih.gov/books/NBK526013/ "Untreated lesions heal spontaneously over 3 to 6 months leaving hyperpigmentation"
- infantile hemangioma https://omim.org/entry/602089 "Hemangiomas were defined as lesions that were present at birth or shortly thereafter, displayed a proliferative growth phase, and showed evidence of regression subsequent to infancy"
